examples/chip-tool/README.md

```diff
@@ -159,23 +159,25 @@ The client will send a single command packet and then exit.
 2. Add Group to device
 
     ```
+    $ chip-tool groups add-group GroupId GroupName node-id endpoint-id
     $ chip-tool groups add-group 0x4141 Light 1234 1
     ```
 
 3. Add group Keyset to device
 
     ```
+    $ chip-tool groupkeymanagement key-set-write GroupKeySet node-id endpoint-id
     $ chip-tool groupkeymanagement key-set-write '{"groupKeySetID": 42,
     "groupKeySecurityPolicy": 0, "epochKey0":
     "d0d1d2d3d4d5d6d7d8d9dadbdcdddedf", "epochStartTime0": 2220000,"epochKey1":
     "d1d1d2d3d4d5d6d7d8d9dadbdcdddedf", "epochStartTime1": 2220001,"epochKey2":
-    "d2d1d2d3d4d5d6d7d8d9dadbdcdddedf", "epochStartTime2": 2220002, }' 1234 0
+    "d2d1d2d3d4d5d6d7d8d9dadbdcdddedf", "epochStartTime2": 2220002 }' 1234 0
     ```
 
 4. Bind Key to group
     ```
-    $ chip-tool groupkeymanagement write group-key-map
-    '[{"groupId": 16705, "groupKeySetID": 42, "fabricIndex": 0}]' 1234 0
+    $ chip-tool groupkeymanagement write group-key-map attr-value node-id endpoint-id
+    $ chip-tool groupkeymanagement write group-key-map '[{"groupId": 16705, "groupKeySetID": 42}]' 1234 0
```

Comment:
`"fabricIndex": 0` was removed from the doc. is it not needed anymore ?

Reply:
Yes, Boris informed me this is no longer necessary

Reply:
Nice work, I've missed that one!